modules/ROOT/pages/index.adoc

```diff
@@ -3,7 +3,7 @@
 == Getting ownCloud
 
 You can download and install ownCloud on your own Linux server, try some prefab cloud or virtual machine images, or sign up for hosted ownCloud services. 
-See the xref:https://owncloud.org/install/[Get Started] page for more information.
+See the link:https://owncloud.org/install/[Get Started] page for more information.
```

Comment:
I did some checking around this because its use was bugging me and found that prefixing external links with `link:` is only necessary "if the target is wrapped". This is according to @mojavelinux (the asciidoctor project lead, and Antora co-lead). Given that, I strongly suggest removing it, unless absolutely necessary.

Reply:
Correct. The link: prefix is only needed when the target is not a URI. That's because the URI protocol is an implicit macro (in other words, https: is recognized as a macro prefix of an implicit link).

Reply:
thanks for clarification :smile: